src/content/1.7/testing/web-acceptance-tests/how-to-contribute-and-create-web-acceptance-tests.md

```diff
@@ -0,0 +1,212 @@
+---
+title: How to contribute and create web acceptance tests
+menuTitle: Contributing to the creation of web acceptance tests
+weight: 2
+---
+
+## How to contribute and create web acceptance tests
```

Comment:
Your header hierarchy is wrong, this should be an H1, the following header an H2, and so on

```suggestion
# How to contribute and create web acceptance tests
```